sync_diff_inspector/diff.go

```diff
@@ -136,7 +143,10 @@ func (df *Diff) AdjustTableConfig(cfg *Config) error {
 	for _, schemaTables := range cfg.Tables {
 		df.tables[schemaTables.Schema] = make(map[string]*TableConfig)
 		tables := make([]string, 0, len(schemaTables.Tables))
-		allTables := allTablesMap[schemaName(df.targetDB.InstanceID, schemaTables.Schema)]
+		allTables, ok := allTablesMap[df.targetDB.InstanceID][schemaTables.Schema]
+		if !ok {
+			return errors.NotFoundf("schema %s.%s", df.targetDB.InstanceID, schemaTables.Schema)
+		}
```

Comment:
can we move this block outer from this `for` loop?

Reply:
seems can't move out